app/views/content/how-to-apply-for-teacher-training/teacher-training-interview.md

```diff
@@ -94,6 +94,8 @@ Providers will also be expecting you to demonstrate you've got relevant skills,
 
 Many providers will ask you to do some form of presentation as they want to see you have the energy and passion to capture pupils' attention. Remember this is a chance to sell yourself and prove your commitment.
 
+$ske$
```

Comment:
I don't think this is appropriate here - SKE isn't something you would do before an interview on a voluntary basis; it is something a provider suggests you need?

I would suggest removing this and lines 24-27